native/app/components/ListPanel.js

```diff
@@ -85,25 +78,25 @@ class ListPanel extends React.Component {
 
   componentDidMount() {
     AppState.addEventListener('change', this._handleAppStateChange);
-    NetInfo.addEventListener('connectionChange', this._handleNetworkStateChange);
+    NetInfo.addEventListener(this._handleNetworkStateChange);
   }
 
   componentWillUnmount() {
     AppState.removeEventListener('change', this._handleAppStateChange);
-    NetInfo.removeEventListener('connectionChange', this._handleNetworkStateChange);
+    NetInfo.removeEventListener(this._handleNetworkStateChange);
   }
 
   shouldComponentUpdate(nextProps) {
     let shouldUpdate = false;
 
-    const changedProps = _.reduce(this.props.state, function (result, value, key) {
-      return _.isEqual(value, nextProps.state[key])
+    const changedProps = lodash.reduce(this.props.state, function(result, value, key) {
+      return lodash.isEqual(value, nextProps.state[key])
         ? result
-        : result.concat(key)
+        : result.concat(key);
     }, []);
 
     // only render when notes collection changes
-    if (changedProps.length > 0 && _.intersection(changedProps, ['notes', 'appUpdates', 'kinto']).length > 0) {
+    if (changedProps.length > 0 && lodash.intersection(changedProps, ['notes', 'appUpdates', 'kinto']).length > 0) {
```

Comment:
IMHO, lodash could even be removed here by some Vanilla JS, but I won't change it here as it doesn't really matter